### PR TITLE
fix: Badge display error when used in menu

### PR DESCRIPTION
**Pull request by iDestin**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

hi! i fixed the issue #35795 

### 💡 Background and solution

image

image

image

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      adjust badge `color` and `line-height`     |
| 🇨🇳 Chinese |     调整徽章的颜色和 `line-height`     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
1. 加个 debug 演示，不修改现有的。
2. next 分支里是 css-in-js 的写法，也帮忙同步一下。

**Comment by iDestin** [comment]
> 1. 加个 debug 演示，不修改现有的。
> 2. next 分支里是 css-in-js 的写法，也帮忙同步一下。

大佬，已经在next分支提了一个pr

**Comment by iDestin** [comment]
#35837

**Comment by afc163** [comment]
1. 冲突了
2. CI 报错

**Comment by iDestin** [comment]
> 1. 冲突了
> 2. CI 报错

冲突是在文档中的演示示例，已解决

**Comment by afc163** [comment]
图片

**Comment by iDestin** [comment]
> 图片

这个我昨晚看了，剩下的这个两个不是由我的改动造成，然后就没改

File: components/menu/style/index.less [hunk 1]
```diff
@@ -247,6 +261,15 @@
     &:focus-visible {
       .accessibility-focus();
     }
+
+    //https://github.com/ant-design/ant-design/issues/35795
```
Comment (afc163) [inline components/menu/style/index.less:265]:
```suggestion
    // https://github.com/ant-design/ant-design/issues/35795
```

File: components/menu/style/index.less [hunk 2]
```diff
@@ -247,6 +261,15 @@
     &:focus-visible {
       .accessibility-focus();
     }
+
+    // https://github.com/ant-design/ant-design/issues/35795
+    > .@{ant-prefix}-badge {
+      color: rgba(255, 255, 255, 0.65);
```
Comment (afc163) [inline components/menu/style/index.less:267]: 最好用现有变量。

Comment (iDestin) [reply components/menu/style/index.less:267]: 好像没有发现这个变量😂，大佬可以指点一下是那个吗

Comment (iDestin) [reply components/menu/style/index.less]: 已改

**Comment by afc163** [comment]
Trace #35837

**Pull request closed without merging**